lib/queue.rb

```diff
@@ -1,28 +1,29 @@
 class Queue
```

Comment:
This definitely passes the tests, but the assignment calls for you to implement the Queue with a circular buffer like we discussed in class.  So, not what we expected

Reply:
Refactored this using the Circular Buffer method. Thanks